packages/documentation/src/stories/components/input/input.docs.mdx

```diff
@@ -51,6 +50,19 @@ But note that the `<input>` element must come first, so we can ensure the correc
 
 ### Sizing
 
+<div className="alert alert-warning">
+  <h4 className="alert-heading">
+    We're deprecating the regular and medium sizes for text input, textarea and select.
+  </h4>
+  <div className="mt-0">
+    This will make it easier to select the appropriate size variant: small for internal
+    applications, large for external applications.
+    <br />
+    `.form-control-rg` and `.form-control-md` are deprecated and will be removed in the next major
+    version.
+  </div>
+</div>
```

Comment:
It's important to have those text elements not on a separate line, because the compiler will wrap them with a `<p>` tag then.

Please make sure to update this everywhere where you added such an alert element!

```suggestion
{/* prettier-ignore */}
<div className="alert alert-warning">
  <h4 className="alert-heading">We're deprecating the regular and medium sizes for text input, textarea and select.</h4>
  <p className="mt-0">This will make it easier to select the appropriate size variant: small for internal applications, large for external applications.<br />
    `.form-control-rg` and `.form-control-md` are deprecated and will be removed in the next major version.</p>
</div>
```